configure

```diff
@@ -4222,26 +4249,39 @@ if test "$_sdlnet" = auto ; then
 	# Check for SDL_Net
 	echocheck "SDL_Net"
 	_sdlnet=no
+
 	cat > $TMPC << EOF
-#include "SDL_net.h"
+#include "SDL3/SDL_net.h"
```

Comment:
Similar thing, I recommend putting SDL3 down the detection list, so it will require special `configure` command line option